Confirmed — I checked the job logs rather than taking it on trust, and your diagnosis holds. Both `Test (ubuntu-latest, 3.13, shard 1)` (job 95418942330) and `Pyright strict` (job 95418942193) fail identically at `Set up job`, before a single test or check runs:

```
Download action repository 'step-security/harden-runner@b09bb98e...'
Failed to download action ... 503 (Service Unavailable)
Failed to download action ... 502 (Bad Gateway)
##[error]Failed to download archive ... after 3 attempts.
```

Worth recording why it looked selective rather than total: codeload was rate-limiting, not down, so each job independently lost or won on its three retries. That is why shards passed and failed in the same run — which is normally the signature of a real failure, and is what made me go and check.

Cleared around 16:20 by the look of it — jobs from then on are pulling the action fine. Nothing needed on your diff.

I tried to re-run the failed jobs and the API refused it for my token (`403 Resource not accessible by integration`), so the re-run needs to come from you or a push rather than from me.

---
_Generated by [Claude Code](https://claude.ai/code)_

--- a/.github/workflows/docs-requirements-staleness-weekly.yml
+++ b/.github/workflows/docs-requirements-staleness-weekly.yml
@@ -75,57 +75,71 @@ jobs:
         if: steps.staleness.outputs.stale == '1'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository }}
         run: |
           set -euo pipefail
           TITLE="docs/requirements.txt transitive pins are behind a fresh resolve"
-          BODY="$(cat <<EOF
-          $(cat staleness.txt)
+          BODY_FILE=issue-body.md
+          {
+            cat staleness.txt
+            echo
+            echo "Taking these is a judgement call rather than a mechanical update, so this"
+            echo "job reports instead of opening a pull request carrying the resolution."
+            echo
+            echo "Workflow run: ${{ github.server_url }}/${REPO}/actions/runs/${{ github.run_id }}"
+            echo
+            echo "Refs #4001."
+          } > "${BODY_FILE}"
 
-          Taking these is a judgement call rather than a mechanical update, so this
-          job reports instead of opening a pull request carrying the resolution.
+          # One perpetual issue, edited in place. The title is fixed rather than
+          # keyed on the moved set: a content-keyed title would close this issue
+          # and open a new one whenever a package drifts, and a closed issue
+          # reads as "resolved" when nothing was resolved. The set moving is the
+          # state persisting, not the state clearing.
+          #
+          # Exact title match, not the search relevance order: `--search` is
+          # fuzzy, so taking `.[0]` could edit a human-authored issue that merely
+          # scored well.
+          EXISTING=$(gh issue list --repo "${REPO}" --state open \
+            --search "in:title \"${TITLE}\"" \
+            --json number,title \
+            --jq ".[] | select(.title == \"${TITLE}\") | .number" \
+            | head -1)
 
-          Workflow run: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
-
-          Refs #4001.
-          EOF
-          )"
-          # One tracking issue, refreshed. A fresh issue every Monday against
-          # a set of pins nobody has decided on yet turns the report into its
-          # own noise source, which is the shape this series keeps removing.
-          EXISTING=$(gh issue list \
-            --repo "${{ github.repository }}" \
-            --search "in:title docs/requirements.txt transitive pins" \
-            --state open \
-            --json number \
-            --jq '.[0].number // ""')
-          if [ -n "$EXISTING" ]; then
-            echo "Refreshing existing tracking issue #$EXISTING"
-            gh issue edit "$EXISTING" \
-              --repo "${{ github.repository }}" \
-              --title "$TITLE" --body "$BODY"
+          if [ -n "${EXISTING}" ]; then
+            echo "Updating existing tracking issue #${EXISTING}"
+            gh issue edit "${EXISTING}" --repo "${REPO}" --body-file "${BODY_FILE}"
           else
-            gh issue create \
-              --repo "${{ github.repository }}" \
-              --title "$TITLE" \
-              --body "$BODY" \
-              --label dependencies \
-              --label ci
+            # Probe the labels first. A blind `create --label x || create`
+            # retries on ANY failure, including a transport error that already
+            # created the issue, and can open a duplicate.
+            LABELS=$(gh label list --repo "${REPO}" --limit 100 --json name --jq '.[].name')
+            LABEL_ARGS=()
+            for candidate in dependencies ci; do
+              if grep -qxF "${candidate}" <<<"${LABELS}"; then
+                LABEL_ARGS+=(--label "${candidate}")
+              fi
+            done
+            gh issue create --repo "${REPO}" \
+              --title "${TITLE}" \
+              --body-file "${BODY_FILE}" \
+              "${LABEL_ARGS[@]}"
           fi
 
       - name: Close the tracking issue when the pins catch up
         if: steps.staleness.outputs.stale == '0'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository }}
         run: |
           set -euo pipefail
-          EXISTING=$(gh issue list \
-            --repo "${{ github.repository }}" \
-            --search "in:title docs/requirements.txt transitive pins" \
-            --state open \
-            --json number \
-            --jq '.[0].number // ""')
-          if [ -n "$EXISTING" ]; then
-            gh issue close "$EXISTING" \
-              --repo "${{ github.repository }}" \
-              --comment "A fresh resolve now matches every committed pin. Closed by ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}."
+          TITLE="docs/requirements.txt transitive pins are behind a fresh resolve"
+          EXISTING=$(gh issue list --repo "${REPO}" --state open \
+            --search "in:title \"${TITLE}\"" \
+            --json number,title \
+            --jq ".[] | select(.title == \"${TITLE}\") | .number" \
+            | head -1)
+          if [ -n "${EXISTING}" ]; then
+            gh issue close "${EXISTING}" --repo "${REPO}" \
+              --comment "A fresh resolve now matches every committed pin. Closed by ${{ github.server_url }}/${REPO}/actions/runs/${{ github.run_id }}."
           fi

--- a/docs/operations/ci-topology.md
+++ b/docs/operations/ci-topology.md
@@ -40,6 +40,7 @@ after workflow changes merge and opens a squash auto-merge PR when the committed
 | .github/workflows/dependency-review.yml | Dependency Review | pull_request | {"cancel-in-progress": "true", "group": "dependency-review-${{ github.event.pull_request.number \|\| github.ref }}"} | 1 |
 | .github/workflows/docs-drift.yml | docs-drift | pull_request, push, schedule, workflow_dispatch | {"cancel-in-progress": "true", "group": "docs-drift-${{ github.ref }}"} | 2 |
 | .github/workflows/docs-observability-snapshot.yml | Observability snapshot | workflow_dispatch | {"cancel-in-progress": "false", "group": "docs-observability-snapshot"} | 1 |
+| .github/workflows/docs-requirements-staleness-weekly.yml | docs-requirements-staleness-weekly | schedule, workflow_dispatch | {"cancel-in-progress": "false", "group": "docs-requirements-staleness-${{ github.ref }}"} | 1 |
 | .github/workflows/eval-weekly.yml | eval-weekly | schedule, workflow_dispatch | {"cancel-in-progress": "false", "group": "eval-weekly-${{ github.ref }}"} | 3 |
 | .github/workflows/feature-matrix-drift.yml | Feature matrix drift | pull_request, push | {"cancel-in-progress": "true", "group": "feature-matrix-drift-${{ github.event.pull_request.number \|\| github.ref }}"} | 1 |
 | .github/workflows/hotfix-r-tracker.yml | Hotfix R-counter | push | {"cancel-in-progress": "false", "group": "hotfix-r-tracker-${{ github.sha }}"} | 1 |
@@ -107,6 +108,7 @@ after workflow changes merge and opens a squash auto-merge PR when the committed
 | .github/workflows/dependency-review.yml | review: Dependency review |
 | .github/workflows/docs-drift.yml | drift-check: Run drift check<br>drift-publish: Publish drift surfaces |
 | .github/workflows/docs-observability-snapshot.yml | snapshot: Capture snapshot |
+| .github/workflows/docs-requirements-staleness-weekly.yml | staleness: recompile and diff |
 | .github/workflows/eval-weekly.yml | bench: bench (full)<br>preflight: preflight (gate)<br>smoke: smoke (synthetic) |
 | .github/workflows/feature-matrix-drift.yml | matrix-rows: registered CLI commands have a matrix row |
 | .github/workflows/hotfix-r-tracker.yml | track: Detect hotfix-begets-hotfix |
@@ -174,6 +176,7 @@ after workflow changes merge and opens a squash auto-merge PR when the committed
 | .github/workflows/dependency-review.yml | workflow: {"contents": "read"}<br>review: {"contents": "read", "pull-requests": "write"} | - |
 | .github/workflows/docs-drift.yml | workflow: {"contents": "read"}<br>drift-check: {"contents": "read"}<br>drift-publish: {"contents": "read", "issues": "write", "pull-requests": "write"} | - |
 | .github/workflows/docs-observability-snapshot.yml | workflow: {"contents": "read"}<br>snapshot: {"contents": "write", "pull-requests": "write", "security-events": "read"} | BERNSTEIN_AUTOSYNC_TOKEN, GITHUB_TOKEN |
+| .github/workflows/docs-requirements-staleness-weekly.yml | workflow: {"contents": "read"}<br>staleness: {"contents": "read", "issues": "write"} | GITHUB_TOKEN |
 | .github/workflows/eval-weekly.yml | workflow: {"contents": "read"} | EVAL_ENABLED |
 | .github/workflows/feature-matrix-drift.yml | workflow: {"contents": "read"}<br>matrix-rows: {"contents": "read"} | - |
 | .github/workflows/hotfix-r-tracker.yml | track: {"contents": "read", "issues": "write", "pull-requests": "write"} | - |

